emva1288/__init__.py

```diff
@@ -1 +0,0 @@
-from ._version import version as __version__
```

Comment:
We migrated to use setuptools_scm in `pyproject.toml` https://github.com/EMVA1288/emva1288/blob/69a2e750967a4edcd242a5009c8dc23f72dd351c/pyproject.toml#L8
The `_version` imported is generated automatically when the package is installed (you have to use the latest pip)
Maybe we should add a comment about that